smartcontracts/sandbox/contracts/W3bstreamRouter.sol

```diff
@@ -21,40 +21,18 @@ contract W3bstreamRouter is IRouter, Initializable {
         fleetManager = _fleetManager;
     }
 
-    function register(uint256 _projectId, address _receiver) external override {
```

Comment:
For security purpose, I think we maybe need project register their receiver first.